src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(operation, number) {
+      this.result = operation(this.result, number);
```

Comment:
you can just execute the callback (operation in your case) on a number
after you adjust methods a bit (see further comments)
```suggestion
      operation(number);
```